packages/@okta/i18n/src/properties/login.properties

```diff
@@ -932,3 +932,8 @@ oie.feature.disabled = The requested feature is not enabled in this environment.
 # Consent
 oie.consent.admin.title = would like to access:
 oie.consent.enduser.title = would like to:
+oie.consent.enduser.accept.label = Yes, it's me
+oie.consent.enduser.deny.label = No, it's not me
+oie.consent.enduser.title = Did you just try to sign in?
+oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
```

Comment:
This is email specific, hence the key is intended to have `email` in it. The rest of the new properties need not be specific to email.